site/docs/v1/tech/apis/_form-request-body.adoc

```diff
@@ -5,11 +5,44 @@
 [field]#form.data# [type]#[Object]# [optional]#Optional#::
 An object that can hold any information about the Form that should be persisted.
 
+[field]#form.type# [type]#[String]# [optional]#Optional# [default]#defaults to `registration`# [since]#Available since 1.20.0#::
+The type of form being created, a form type cannot be changed after the form has been created. This type will be used to identify how this form can be utilized by FusionAuth.
++
+ * `registration` This form will be used for self service registration.
+ * `adminRegistration` This form be used to customize the add and edit User Registration form in the FusionAuth UI.
+ * `adminUser` This form can be used to customize the add and edit User form in the FusionAuth UI.
+
+
 [field]#form.name# [type]#[String]# [required]#Required#::
 The unique name of the Form.
 
 [field]#form.steps# [type]#[Array<Object>]# [required]#Required#::
 An ordered list of objects containing one or more Form Fields. A Form must have at least one step defined.
++
+Steps manifest in two different ways. When the form type is `adminRegistration` or `adminUser` steps will be rendered as `sections` because there are no multi-step forms within the UI. For `registration` each step will be considered a step within a multi-step form.
++
+*Localization*
++
+You may localize the displayed value of this key during registration by adding the key to your theme.
+For example, if you specify the key value of `{user-form-section}1` this key value be rendered in the UI, where `1` specifies the section number to adjust the value of.
++
+If you were to add the following messages to your theme, the localized form of the value will be shown during the administrator registration. This allows you to provide a customized section name and optionally localize the field name as well.
++ 
+```
+{user-form-section}1=Optionally name me!
+```
++
+If you utilize the same form for multiple tenants, you can specify the specific configuration for the tenant by providing the unique Id for the tenant. An example would be as follows:
++
+```
+{user-form-section.cbeaf8fe-f4a7-4a27-9f77-c609f1b01856}2=Tenant specific label for section 2
+```
++
+If you would like to configure a form to have different section names based on an application, you can provide the unique Id for the application. An example would be as follows:
++
+```
+{registration-form-section.cfb5fab7-b3b6-41bb-adfa-d23ac83a96e5}2=Application specific label for section 2
```

Comment:
I did end up committing that code to change how we are doing the tenant specific stuff. 

So the new pattern is:
```
[cbeaf8fe-f4a7-4a27-9f77-c609f1b01856]{user-form-section}1=Tenant specific label for section 2
```